plugin/src/test/java/io/jenkins/plugins/casc/misc/Util.java

```diff
@@ -1,38 +1,142 @@
 package io.jenkins.plugins.casc.misc;
 
+import hudson.ExtensionList;
 import io.jenkins.plugins.casc.ConfigurationAsCode;
 import io.jenkins.plugins.casc.ConfigurationContext;
 import io.jenkins.plugins.casc.core.JenkinsConfigurator;
+import io.jenkins.plugins.casc.impl.configurators.GlobalConfigurationCategoryConfigurator;
 import io.jenkins.plugins.casc.model.CNode;
 import io.jenkins.plugins.casc.model.Mapping;
 import io.jenkins.plugins.casc.snakeyaml.nodes.Node;
+import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.StringWriter;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.Objects;
+import jenkins.model.GlobalConfigurationCategory;
 import jenkins.model.Jenkins;
 
 import static io.jenkins.plugins.casc.ConfigurationAsCode.serializeYamlNode;
 
 public class Util {
+
+    /**
+     * Gets the jenkins configurator
+     * @return the jenkins configurator
+     */
     public static JenkinsConfigurator getJenkinsConfigurator() {
         return Jenkins.getInstance().getExtensionList(JenkinsConfigurator.class).get(0);
     }
 
+    /**
+     * Gets the JenkinsRoot Mapping <br/>
+     *
+     * Example usage:
+     * <pre>{@code
+     * ConfiguratorRegistry registry = ConfiguratorRegistry.get();
+     * ConfigurationContext context = new ConfigurationContext(registry);
+     * final CNode configNode = getJenkinsRoot(context).get("nodes");
+     * }</pre>
+     *
+     * @param context the configuration context
+     * @return the jenkins root mapping
+     * @throws Exception something's not right...
+     */
     public static Mapping getJenkinsRoot(ConfigurationContext context)
         throws Exception {
         JenkinsConfigurator root = getJenkinsConfigurator();
         return Objects.requireNonNull(root.describe(root.getTargetComponent(context), context)).asMapping();
     }
 
-    public static Mapping getJenkinsRoot(JenkinsConfigurator root, ConfigurationContext context)
```

Comment:
I couldn't see any reason for anyone to use this? and there doesn't seem to be any usages of it?

Can restore it if you see a need

Reply:
Fine with me